From bb934833d1621ff50ca68cc9f21bbec1e90255b4 Mon Sep 17 00:00:00 2001
From: Evgeny Grin <k2k@narod.ru>
Date: Sun, 21 May 2023 16:16:24 +0300
Subject: [PATCH] Remove duplicated libs from MCLIBS (again)

Fixed copy-paste error.
This is a correction for 92ae4c6b2027315d32059e8fed2c601a0dff1070.
---
 m4.include/mc-with-screen-ncurses.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4.include/mc-with-screen-ncurses.m4 b/m4.include/mc-with-screen-ncurses.m4
index 5411ff032..843d044a5 100644
--- a/m4.include/mc-with-screen-ncurses.m4
+++ b/m4.include/mc-with-screen-ncurses.m4
@@ -122,7 +122,7 @@ AC_DEFUN([mc_WITH_NCURSES], [
             MCLIBS="$LIBS"
         fi
         LIBS="$MCLIBS"
-        AC_SEARCH_LIBS([stdscr], [tinfow tinfo], [MCLIBS="$MCLIBS $LIBS"],
+        AC_SEARCH_LIBS([stdscr], [tinfow tinfo], [],
                        [AC_MSG_ERROR([Cannot find a library providing stdscr])])
         MCLIBS="$LIBS"
 
-- 
2.40.1.windows.1

